src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -503,7 +506,11 @@
                                         "items": {"$ref": "#/definitions/stream_name"},
                                         "uniqueItems": true
                                     },
-                                    "w_cutoff": {"$ref": "#/definitions/positive_number", "default": 1.0},
+                                    "w_cutoff": {"$ref": "#/definitions/nonneg_number", "default": 1.0},
+{% if version >= "4.4" %}
+                                    "window_function": {"$ref": "#/definitions/window_function"},
+                                    "taps": {"$ref": "#/definitions/positive_integer"},
```

Comment:
Admittedly not the most pressing question, just while we're here
* Any particular reason `w_cutoff` has a default but `window_function` and `taps` do not?

I know there are defaults in 
* `katsdpcontroller/product_config.py::GpuCbfAntennaChannelisedVoltage` parameter list, and in
* `fgpu/main.py` for `parse_{wideband, narrowband}`

Reply:
Good question. I should probably add the default for window_function. I don't want to add one for taps, because that's implementation-defined rather than part of the schema itself.

It's a fairly arbitrary choice, because the default values defined here have no impact and serve purely as documentation.